index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-214417857-1"></script>
+	<script>
+	  window.dataLayer = window.dataLayer || [];
+	  function gtag(){dataLayer.push(arguments);}
+	  gtag('js', new Date());
+
+	  gtag('config', 'UA-214417857-1');
+	</script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" media="(min-width:992px)" href="desktop.css">
+    <link rel="stylesheet" media="(max-width:992px)" href="style.css">
+    <title>Phil's Portfolio</title>
+</head>
+<body>
+  <header class="main-section">
+    <section class="toolbar">
+      <nav class="desktop">
+        <div class="all-item"><a href="#">My Logo</a></div>
+        <div class="desk-item">
+          <div><a href="#portfolio">Portfolio</a></div>
+          <div><a href="#about">About</a></div>
+          <div><a href="#contact">Contact</a></div>
+        </div>
+        <div class="mobile-item">
+          <div class="item close"><a href="#" class="closeMenu"><i class="fa fa-times" aria-hidden="true"></i></a></div>
+          <div class="mobile-nav"><a href="#portfolio" class="closeMenu">Portfolio</a></div>
+          <div class="mobile-nav"><a href="#about" class="closeMenu">About</a></div>
+          <div class="mobile-nav"><a href="#contact" class="closeMenu">Contact</a></div>
+        </div>
+        <div class="item menu"><a href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></a></div>
+      </nav>
+    </section>
+  </header>
+  <main class="main-parts">
+    <section class="headline-section">
```

Comment:
@dumbledorep great work implementing the changes requested.

Reply:
@danmainah Thank you